content/en/blog/_posts/2024-04-02-introducing-jobset/2024-04-02-introducing-jobset.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog
+title: "Introducing JobSet"
+date: 2024-04-02
+slug: introducing-jobset
+---
+
+**Authors**: Daniel Vega-Myhre (Google), Abdullah Gharaibeh (Google), Kevin Hannon (Red Hat)
+
+In this article, we introduce [JobSet](https://jobset.sigs.k8s.io/), an open source API for representing distributed jobs. The goal of JobSet is to provide a stable API for running/building APIs with AI/ML and HPC use cases in mind.
+
+## Why JobSet?
+
+The Kubernetes community’s recent enhancements to the batch ecosystem on Kubernetes attracted ML engineers who have found it to be a natural fit for the requirements of running distributed training workloads. Large ML models (particularly LLMs) which cannot fit into the memory of the GPU or TPU chips on a single host are often distributed across tens of thousands of accelerator chips, which in turn may span thousands of hosts. 
+
+As such, the model training code is often containerized and executed simultaneously on all these hosts, performing distributed computations which often shard both the model parameters and/or the training dataset across the target accelerator chips, using NCCL or XLA collective primitives like all-gather and all-reduce to perform distributed computations and synchronize gradients between hosts. 
```

Comment:
Please define definitions. NCCL or XLA is well known enough for the general audience.

Reply:
How about making NCCL and XLA links to the docs? 

Otherwise the sentence may become a bit hard to follow:

`... using NCCL (library implementing multi-GPU and multi-node communication primitives) or XLA (compiler targeting XLA devices, such as TPUs, including collective primitives for multi-TPU and multi-node communication) collective primitives like all-gather and all-reduce to perform...`

Reply:
True, maybe it can just be:

`... using collective primitives like all-gather and all-reduce to perform...`

Reply:
SGTM.